deploy.sh

```diff
@@ -152,8 +152,12 @@ svn update
 
 svn status
 
-echo "➤ Committing files..."
-svn commit -m "Update to version $VERSION from GitHub" --no-auth-cache --non-interactive  --username "$SVN_USERNAME" --password "$SVN_PASSWORD"
+if [[ "$DEBUG" != false ]]; then
+  echo "➤ Committing files..."
+  svn commit -m "Update to version $VERSION from GitHub" --no-auth-cache --non-interactive  --username "$SVN_USERNAME" --password "$SVN_PASSWORD"
+else
+  echo "➤ Debug mode: Files not committed."
```

Comment:
I think this is the point where @joemcgill is looking for some sort of output to view what _would_ have been sent across to SVN